### DISCUSSION TITLE
RSS feed for all comments

### DISCUSSION THREAD

**Discussion by cambid** [posativ/isso#914] [Ideas]
There is no rss feed for all articles but only for all the comments to one article, right? Wouldn't it be great to provide a feed for all articles which can be included in the head of websites/blogs.

EDIT: To clarify: I am looking for a feed of all comment of all articles not a feed with all articles.

**Comment by ix5** [comment]
There is the `/latest` [API endpoint](https://isso-comments.de/docs/api/#api-Comment-latest), but no catch-all RSS feed.
Might be an idea to implement, but to keep disabled by default for privacy reasons.

**Comment by jelmer** [comment]
isso itself doesn't have the articles, so it would be hard to generate a full RSS feed that includes the articles - we could only include the comments on the articles.

**Reply by cambid** [reply]
Thanks for your comment. I'm not searching for a feed of all articles. This is provided by the used SSG (in my example hugo). But I am looking for a feed with the comments to all articles.

Please take my apology for the misleading starting in the thread.

**Reply by jelmer** [reply]
I think the use of the word articles here is misleading; aren't you just looking for a feed of all comments? How is this different from /latest ?

**Reply by Guts** [reply]
`/latest` is JSON, not RSS (or Atom or JSON Feed).

If I got the point of, it's about to propose a RSS feed for all comments. The title is indeed misleading. @cambid can you edit it if you agree?

**Reply by cambid** [reply]
Yes I'm looking for an RSS feed of all the comments.

**Comment by Guts** [comment]
I'm definitely interested in this feature.
